lib/rules/validate-jsdoc.js

```diff
@@ -230,24 +230,42 @@ function patchNodesInFile(file) {
      */
     function getJsdoc() {
         if (!this.hasOwnProperty('_jsdoc')) {
-            var res = findDocCommentBeforeLine(this.loc.start.line);
+            var node = (this.type === 'FunctionExpression') ? findFirstNodeInLine(this) : this;
+            var res = findDocCommentBeforeNode(node);
```

Comment:
Thanks for contribution! But... could you try to find `getPrevToken(..., { include: 'comments' })` with the new jsfile api? https://github.com/jscs-dev/node-jscs/blob/master/lib/js-file.js#L290 ;-) Sorry for that